examples/bastion-host/main.tf

```diff
@@ -1 +1,47 @@
+module "management_network" {
+  # When using these modules in your own templates, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-google-network.git//modules/vpc-network?ref=v0.1.0"
```

Comment:
Should this match the ref tag below @ v0.1.1?

Reply:
Bumping to 0.1.2 since I'll make a new release including this.